db/seeds.rb

```diff
@@ -5,3 +5,44 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```

Comment:
Es bueno siempre agregar un algún comentario el seed como por ejemplo:
```suggestion
puts 'creando actividades...'
```

Reply:
Hecho